### PR TITLE
Release candidate changes

### PR DESCRIPTION
**Pull request by imaji**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,14 +225,9 @@ else {
 
 * Developed using Visual Studio Community 2013
 * The NUnit test framework is used for testing, your copy of Visual Studio needs the "NUnit test adapter" installed from Tools -> Extensions and Updates if you wish to run the test from the IDE.
-* PusherServer acceptance tests depends on [PusherClient](https://github.com/pusher/pusher-dotnet-client).
+* PusherServer acceptance tests depends on [PusherClient](https://github.com/leggetter/pusher-dotnet-client).
```
Comment (ewmy) [inline README.md]: should be https://github.com/pusher-community/pusher-websocket-dotnet

File: README.md [hunk 2]
```diff
@@ -158,7 +162,7 @@ pusher.FetchStateForChannelsAsync<object>((ITriggerResult result) =>
 Retrive a list of users that are on a presence channel:
 
 ```
-IGetResult<object> result = pusher.Get<object>("/channels/presence-channel/users" );
+IGetResult<object> result = pusher.FetchUsersFromPrecence<object>("/channels/presence-channel/users" );
```
Comment (ewmy) [inline README.md]: Presence is spelt wrong here

File: README.md (hunk 1 above)
Comment (imaji) [inline README.md]: Corrected

File: README.md (hunk 2 above)
Comment (imaji) [inline README.md]: Corrected

**Pull request merged**